pages/restake/bridge.mdx

```diff
@@ -0,0 +1,78 @@
+import Callout from "/components/Callout";
+
+# Bridge
+
+In order to participate in Tangle's restaking infrastructure, users need to first bridge in their assets from EVM-based networks such as Ethereum into Tangle. For this, we have a dedicated bridge & its corresponding DApp page.
+
+[Access Tangle DApp's Bridge page here](https://app.tangle.tools/bridge)
```

Comment:
I would create a card for this similar to what we have in staking cards. Bigger cards are clear.